cmd/snap/cmd_prepare_image.go

```diff
@@ -40,8 +40,11 @@ type cmdPrepareImage struct {
 
 func init() {
 	cmd := addCommand("prepare-image",
-		i18n.G("Prepare a snappy image"),
-		i18n.G("Prepare a snappy image"),
+		i18n.G("Prepare a snappy device image"),
+		i18n.G(`
+The prepare-image command is used by ubuntu-image in building a snappy device
```

Comment:
"The prepare-image command performs some of the steps necessary when creating core device images."

That avoids special casing the command to a single user.